fv3core/utils/grid.py

```diff
@@ -392,7 +392,7 @@ def compute_origin(self, add: Tuple[int, int, int] = (0, 0, 0)):
         """Start of the compute domain (e.g. (halo, halo, 0))"""
         return (self.is_ + add[0], self.js + add[1], add[2])
 
-    def full_origin(self):
+    def full_origin(self, add: Tuple[int, int, int] = (0, 0, 0)):
         """Start of the full array including halo points (e.g. (0, 0, 0))"""
         return (self.isd, self.jsd, 0)
```

Comment:
The code doesn't actually add the `add` argument. You can either remove the argument if it's not needed like this, or you can fix the code.

Reply:
good catch, done too quickly in response to a review comment